src/Chapter1.hs

```diff
@@ -632,7 +646,11 @@ Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
 
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = lastNum (abs n) + lastNum (abs n `div` 10)
+  where
+    lastNum ::  Int -> Int
+    lastNum x = mod x 10
```

Comment:
Great!
You can also use `lastDigit` function that you implemented above that already using `abs` inside, so you don't need to pass `abs n` but can do just `n` instead 👌🏼

Reply:
I tried this out earlier but it fails for negative number. Seems like GHC evaluates `(-n) % 10` first and then passes it to `x` in `lastNum` here. So without `mod`, it'll produce different result than `abs (-n) % 10` and thus result into unexpected answer. Verified by running tests as well.